ci/deploy_tidb_operator_staging.groovy

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2020. PingCAP, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import groovy.transform.Field
+
+@Field
+def values = '''
+controllerManager:
+  replicas: 2
+scheduler:
+  replicas: 2
+admissionWebhook:
+  create: true
+  hooksEnabled:
+    statefulSets: true
+    pods: true
+    # TODO: enable validating and defaulting after we ease the constrain
+    validating: false
+    defaulting: false
+'''
+
+def call(BUILD_BRANCH) {
+    def GITHASH
+    def UCLOUD_OSS_URL = "http://pingcap-dev.hk.ufileos.com"
+
+    catchError {
+        node('delivery') {
+            container('delivery') {
+                def WORKSPACE = pwd()
+                sh "chown -R jenkins:jenkins ./"
+                deleteDir()
+
+                dir("${WORKSPACE}/operator-staging") {
+                    stage('Download tidb-operator binary') {
+                        GITHASH = sh(returnStdout: true, script: "curl ${UCLOUD_OSS_URL}/refs/pingcap/operator/${BUILD_BRANCH}/centos7/sha1").trim()
+                        sh "curl ${UCLOUD_OSS_URL}/builds/pingcap/operator/${GITHASH}/centos7/tidb-operator.tar.gz | tar xz"
+                    }
+                    stage('Push tidb-operator Docker Image to Harbor') {
+                        docker.withRegistry("https://hub.pingcap.net", "harbor-pingcap") {
+                            sh"""
+                        docker build --network=host -t hub.pingcap.net/jenkins/tidb-operator:${GITHASH} images/tidb-operator
+                        docker push hub.pingcap.net/jenkins/tidb-operator:${GITHASH}
```

Comment:
Refer [here](https://github.com/pingcap/tidb-operator/blob/master/ci/release_tidb_operator_binary_and_image.groovy#L24-L29) to push image?

Reply:
addressed in https://github.com/pingcap/tidb-operator/pull/1505/commits/8003622b4424e40ab00c6f38ef3c05b1aa49939d